pkg/auth/authorizer/abac/abac.go

```diff
@@ -51,13 +60,74 @@ func (p policyLoadError) Error() string {
 
 type policyList []*api.Policy
 
+type ABACPolicy struct {
+	Auth atomic.Value
+}
+
+func New(path string) (*ABACPolicy, error) {
+
+	var plist policyList
+	abacPolicy := &ABACPolicy{}
+
+	plist, err, eof := newFromFile(path)
+	if err != nil {
+		return nil, err
+	}
+
+	if eof {
+		go wait.Forever(func() {
+			var lastModifiedTime *time.Time
+			for {
+				time.Sleep(filePollPeriod)
+
+				done := make(chan error)
+				go func() {
```

Comment:
This doesn't need to be async, but it keeps the code inside of `go func() {...` cleaner, otherwise, for each error condition, we would need to `glog.Warning(...)` and `time.Sleep(failureRetryPeriod)`.